### ISSUE TITLE
Feature to export the document as a Latex file

### ISSUE THREAD

**Issue by sebi5361** [jhuix/vscode-markdown-preview-showdown#9]
>> ... AsciiMath to Latex ... version of the js module ...
>
> As I understand there is an intermediate state before rendering with some  _AsciiMath to Latex_ conversion.
> Thus is there and easy way to offer a feature of exporting the document to a Latex file?

I really don't know how complicated this might be, especially as your viewer supports many charts that might not be convertible to Latex easily. I don't know.

Still this additional feature would be neat!
But it might be over the scope of _vscode-markdown-preview-showdown_ as it is at first a viewer (it has already some export options though).

**Comment by jhuix** [comment]
I'm not sure know what tools are available to convert HTML or PDF file into tex file. Pandoc？ Do you have any good suggestions?

**Comment by sebi5361** [comment]
Giving more thoughts to this, I think the best option would be to offer users a possibility to export `.md` files to, let's call them, `.pandoc-proof.md` files, meaning `.md` files that can be used by pandoc as inputs for further conversions.

What I mean by this, is that I don't see options today in pandoc to convert AsciiMath+markdown files in other formats, as AsciiMath is missing from the conservation possibility list. Thus generating a `.pandoc-proof.md` files where all AsciiMath formulas have been substituted by their Latex equivalent formulas would be a must.

This could also hold for all the graphs your viewer is capable of rendering: Generating a `.pandoc-proof.md` where all graphs have been substituted by links to their rendered pictures,  could be useful to some users.

I am interested by the AsciiMath aspect only today, but the other aspects might interest other people.

I believe internally, translating AsciiMath formulas to Latex formulas is already done in a intermediate step hidden to users.
Indeed, while playing with the delimiter settings I saw that case where entering `deltax` in my markdown file resulted in `\delta x` being displayed this way on the preview panel. I forgot what the delimiter settings I entered at that time , and couldn't figure out how to reproduce this situation, but I am pretty sure I saw it.

So maybe generating that `.pandoc-proof.md` file is not that big of a deal for you and would be a neat feature. Up to us then, the users, to translate that `.pandoc-proof.md` file in the formats we want...
